### PR TITLE
maksym/japan-ui

### PR DESCRIPTION
**Pull request by binary-maksym**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "website_name": "Binary.com",
     "display_languages": "EN,ID,RU,ES,FR,PT,DE,ZH_CN,PL,AR",
     "static": {
-        "url": "https://static.binary.com/",
```
Comment (raunakkathuria) [inline config.json]: remove this change

Comment (binary-maksym) [inline config.json]: its not ready for production. When it is, I will delete this change

File: build/concat.js [hunk 2]
```diff
@@ -13,7 +13,8 @@ module.exports = {
              ],
             'dist/js/binary.js': [
                 'src/javascript/binary/base/*.js',
-                'src/javascript/binary/**/*.js'
+                'src/javascript/binary/**/*.js',
```
Comment (jy-binary) [inline build/concat.js]: what are these **?

File: build/fileindex.js [hunk 3]
```diff
@@ -16,7 +16,8 @@ module.exports = {
             'src/javascript/autogenerated/texts.js',
             'src/javascript/autogenerated/*.js',
             'src/javascript/binary/base/*.js',
-            'src/javascript/binary/**/*.js'
+            'src/javascript/binary/**/*.js',
```
Comment (jy-binary) [inline build/fileindex.js]: here too

File: config.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
...
+        "url": "https://binary-maksym.github.io/binary-static/",
```
Comment (jy-binary) [inline config.json]: can't have binary-maksym.github.io in production

Comment (binary-maksym) [inline config.json]: we will not

File: build/fileindex.js (hunk 3 above)
Comment (binary-maksym) [inline build/fileindex.js]: ** - all folders

**Comment by raunak-binary** [comment]
@borisyankov can you also review this

**Comment by raunakkathuria** [comment]
Can you see the test, they are failing. Will merge after that

**Comment by binary-maksym** [comment]
Sorry, my fault, have fixed

On Mon, Sep 7, 2015 at 4:04 PM, Raunak Kathuria <notifications@github.com>
wrote:

> Can you see the test, they are failing. Will merge after that
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/binary-com/binary-static/pull/433#issuecomment-138222559>
> .
>



-- 
[image: photo] <http://www.facebook.com/catavary>
<http://www.linkedin.com/pub/maksym-kotielnikov/b3/b38/6b2>Maksym
Kotielnikov
Software Engineer, Binary Group Services Sdn. Bhd.
011-3517-2930  <011-3517-2930>// Skype: maxim_kotelnikov

**Pull request merged**